core/themes/bartik/templates/block--layout--hero-block.tpl.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * @file
+ * Template for outputting the default block styling within a Layout.
+ *
+ * Variables available:
+ * - $classes: Array of classes that should be displayed on the block's wrapper.
+ * - $title: The title of the block.
+ * - $title_prefix/$title_suffix: A prefix and suffix for the title tag. This
+ *   is important to print out as administrative links to edit this block are
+ *   printed in these variables.
+ * - $content: The actual content of the block.
+ */
+?>
+<div class="maintain-aspect-ratio <?php print implode(' ', $classes); ?>"<?php print backdrop_attributes($attributes); ?>>
```

Comment:
Actually this markup is so similar... do we really need to wrap the title in `content`? Do you think we could manage without an additional template?